CHANGELOG.md

```diff
@@ -5,8 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
-  - nothing yet
+## Unreleased [3.5.2]
```

Comment:
The Changelog should also include under Added: "Added sanitizer option for detecting memory leaks in HEMCO standalone during build"

Reply:
Now added in commit: 471fa19e470a1b8a8c379305143bdbdaf5bdc945